pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/ForwardIndexConfig.java

```diff
@@ -92,6 +92,7 @@ public static ForwardIndexConfig getDisabled() {
   private final String _targetMaxChunkSize;
   private final int _targetMaxChunkSizeBytes;
   private final int _targetDocsPerChunk;
+  private final Map<String, Object> _mapEncodingConfigs;
```

Comment:
IIUC, map index is a collection of different types of indices. We should probably add it as a separate index type